### PR TITLE
added example how to treat exceptions

### PR DESCRIPTION
**Pull request by umbri**
I think this must be somewhere in docs not in examples.

### CODE REVIEW

File: examples/exceptions/exception.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+require_once __DIR__ . "/../../vendor/autoload.php";
+
+function asyncOperation() {
+    $def = new Deferred();
+
+    \Amp\Loop::delay(1000, function () use ($def) {
+        $def->fail(new Exception("force exception")); //this will be catch-ed
+
+//        throw new Exception("force exception"); //this will be NOT catch-ed
```
Comment (kelunik) [inline examples/exceptions/exception.php]: I think there should be a second async function that throws inside the `delay` callback and a second `Loop::run`.

**Comment by kelunik** [comment]
@umbri Thanks! I added a comment. Maybe you also want to fix your email, so the commit is properly linked with your GitHub account.

File: examples/exceptions/exception.php (hunk 1 above)
Comment (umbri) [reply examples/exceptions/exception.php]: @kelunik OK will add 2 example catch-ed & uncatch-ed and PR

**Comment by umbri** [comment]
@kelunik Thank you! I will fix it

File: examples/exceptions/uncatched.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Loop::run(function () {
+    try {
+        $res = yield asyncOperation();
+
+        echo "result -> " . $res . PHP_EOL;
+    } catch (Throwable $e) {
+        echo "catch -> " . $e->getMessage() . PHP_EOL;
+    }
+});
```
Comment (kelunik) [inline examples/exceptions/uncatched.php]: I'd add a second `try { ... } catch` around the `Loop::run` to show that exceptions thrown in loop callbacks just bubble up.

**Pull request closed without merging**